dimagi/commcare-hq
# Remove Angular form HQ

## Summary

https://dimagi-dev.atlassian.net/browse/SAAS-12329
We are removing Angular from HQ as it is no longer used.
## Safety Assurance


### Automated test coverage



### QA Plan


Not raising a formal QA because there is no evidence of usage of these libraries on HQ. However, they are pushed to staging to observe if we see any related issues.
### Safety story

The changes have been tested on local and the branch is pushed to staging and we would keep it there till Monday to see if someone finds any issues.
### Rollback instructions
